src/views/AgmLocationChg.vue

```diff
@@ -78,45 +60,104 @@
             </header>
 
+            <!-- Help -->
             <ExpandableHelp helpLabel="Help with Annual General Meeting Extension">
               <template #content>
-                Help text, or sub-component, goes here.
+                <section class="agm-location-change-help">
+                  <h3 class="text-center">
+                    AGM Location Change Help
+                  </h3>
+                  <div class="mt-6">
+                    <p class="ml-1">
+                      This request for location change is only needed if the meetings are fully in-person. In general,
+                      these meetings are required to be held in British Columbia. However, there are exceptions to this
+                      rule. If the company's articles permit it or if the members agree, the meeting can be conducted at
+                      a location outside British Columbia. If the meeting combines both in-person and online
+                      participation, the location requirement only applies to the in-person component. If the meeting is
+                      entirely conducted online, these location restrictions do not apply.
+                    </p>
+                  </div>
+                </section>
               </template>
             </ExpandableHelp>
 
             <!-- Main Section -->
             <section class="mt-8">
               <header>
-                <h2>Section Title</h2>
+                <h2>AGM Location Detail</h2>
                 <p class="grey-text">
-                  Section subtext.
+                  Enter the new AGM location detail including any
                 </p>
               </header>
 
-              <div
-                id="main-section"
-                :class="{ 'invalid-section': !sectionValid && showErrors }"
-              >
+              <div>
                 <v-card
                   flat
                   class="py-4 px-5"
                 >
+                  <!-- AGM Year -->
                   <v-row
                     no-gutters
-                    class="my-4"
+                    class="my-6"
                   >
                     <v-col
                       cols="12"
                       sm="3"
                       class="pr-4"
                     >
-                      <strong :class="{ 'app-red': !sectionValid && showErrors }">Subtitle</strong>
+                      <strong :class="{ 'app-red': !agmYearValid && showErrors }">AGM Year</strong>
+                    </v-col>
+                    <v-col
+                      cols="12"
+                      sm="4"
+                    >
+                      <div
+                        id="agm-year-section"
+                        :class="{ 'invalid-section': !agmYearValid && showErrors }"
+                      >
+                        <AgmYear
+                          ref="agmYearRef"
+                          v-model="agmYear"
+                          :label="'AGM year'"
```

Comment:
The colon isn't needed here because the string isn't dynamic. You can just write:
`label="AGM year"`